### PR TITLE
Update docs and fix class scope issue

### PR DESCRIPTION
**Pull request by pglebocki**

### CODE REVIEW

File: app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java [hunk 1]
```diff
@@ -207,8 +207,8 @@ public Builder setContent(String content) {
          * This method sets dialog content description text, which is visible below title.
          * The description content is optional.
          *
-         * @param resId
-         * @return
+         * @param resId resource if of dialog's content text
```
Comment (zawadz88) [inline app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java]: resource if -> resource id ;)

File: app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java [hunk 2]
```diff
@@ -289,8 +289,8 @@ public Builder setTitleColor(@ColorRes int colorResId) {
          * If not set then if uses default primary text color
          * defined in theme.
          *
-         * @param colorResId
-         * @return
+         * @param colorResId color resource if for content label
```
Comment (zawadz88) [inline app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java]: same here ;)

Comment (pglebocki) [inline app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java]: corrected

File: app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java (hunk 1 above)
Comment (pglebocki) [inline app-rating/src/main/java/com/stepstone/apprating/AppRatingDialog.java]: corrected

**Pull request merged**